hack/make-rules/verify/codegen.sh

```diff
@@ -22,12 +22,14 @@ cd $REPO_ROOT
 
 # place to stick temp binaries
 BINDIR="${REPO_ROOT}/_bin"
+mkdir -p "${BINDIR}"
```

Comment:
not sure if we want `-p`, since BINDIR doesn't appear to be overridable, and we should expect REPO_ROOT to exist? if this fails without -p then we have REPO_ROOT wrong and probably want to catch that.

Reply:
when _bin dir already exists, mkdir exits 0 with -p

Reply:
I mean, having `-p` means it will try to create the entire hierarchy if it `$REPO_ROOT/` exist.

Reply:
yeah I can see that, I think I can make it safer by:
```
if [[ ! -d "$BINDIR" ]]; then
  mkdir $BINDIR
fi
```

Reply:
done